src/component/container/LayerTreeClassic/LayerTreeClassic.tsx

```diff
@@ -92,16 +99,27 @@ export class LayerTreeClassic extends React.Component<LayerTreeClassicProps> {
     }
   }
 
+  onHideLayerTree() {
+    this.props.dispatch(hideLayerTree());
```

Comment:
This will hide the tree, but how to enable it again? Maybe it's the easiest to introduce a `toggleTreeVisibility()` dispatcher.

Reply:
I tried `toggleTreeVisibility()` but this results in conflict with the `SimpleToggleButton` in the toolbars (infinite loop). Since this is a close button, I think hiding is enough.